### PR TITLE
Copy update on /ceph

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Copy update on `/ceph`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/ceph
- Be sure to test on mobile, tablet and desktop screen sizes
- Please check page against [copy doc](https://docs.google.com/document/d/1gpEjTcdJor2yTGfMz7T_xIfs7BtV6rbBPFYfNZFe52Y/edit#) and resolve comments

## Issue / Card

Fixes [#10970](https://github.com/canonical-web-and-design/ubuntu.com/issues/10970)

## Screenshots

Ceph storage on Ubuntu | Ubuntu (2021-12-07 09-31-17)

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/ceph/index.html [hunk 1]
```diff
@@ -58,7 +59,7 @@ <h2>Ceph storage done right</h2>
         ) | safe
       }}
       <hr class="u-sv1">
-      <p class="p-card__content">In the early days of Ceph, Yahoo! Japan searched for a local company who could support them in the implementation of an open-source distributed storage solution from the Linux operating system all the way to Ceph. Canonical provided not only Ceph but also the tooling and expertise that reduced the OPEX associated with  managing a large storage cluster.</p>
+      <p class="p-card__content">In the early days of Ceph, Yahoo! Japan searched for a local company who could support them in implementating an open-source distributed storage solution from the Linux operating system all the way to Ceph. Canonical provided not only Ceph but also the tooling and expertise that reduced the OPEX associated with managing a large storage cluster.</p>
```
Comment (anthonydillon) [inline templates/ceph/index.html:62]:
```suggestion
      <p class="p-card__content">In the early days of Ceph, Yahoo! Japan searched for a local company who could support them in implementing an open-source distributed storage solution from the Linux operating system all the way to Ceph. Canonical provided not only Ceph but also the tooling and expertise that reduced the OPEX associated with managing a large storage cluster.</p>
```

File: templates/ceph/index.html [hunk 2]
```diff
@@ -84,10 +85,10 @@ <h2>Ceph storage done right</h2>
 <section class="p-strip--light">
   <div class="u-fixed-width">
     <h2>Ceph storage deployment and operations</h2>
-    <p>Canonical can help you to design and install your Ceph cluster or provide consulting to enable you to run Ceph autonomously. If you are interested in having automated Ceph day 2 operations, you can opt for our offerings of Charmed Ceph, a Ceph cluster encapsulated in Juju charms, or Managed Ceph &mdash; a Ceph cluster fully managed by Canonical.</p>
+    <p>Canonical can help you to design and install your Ceph cluster or provide consulting to enable you to run Ceph autonomously. If you are interested in having automated Ceph Day-2 operations, you can opt for our offerings of Charmed Ceph, a Ceph cluster encapsulated in Juju charms. If you prefer to treat your storage as-a-service, we offer Managed Ceph - a Ceph cluster fully managed by Canonical.</p>
```
Comment (anthonydillon) [inline templates/ceph/index.html:88]:
```suggestion
    <p>Canonical can help you to design and install your Ceph cluster or provide consulting to enable you to run Ceph autonomously. If you are interested in having automated Ceph Day-2 operations, you can opt for our offerings of Charmed Ceph, a Ceph cluster encapsulated in Juju charms. If you prefer to treat your storage as-a-service, we offer Managed Ceph &mdash; a Ceph cluster fully managed by Canonical.</p>
```

File: templates/ceph/index.html [hunk 3]
```diff
@@ -170,7 +178,7 @@ <h4>Encryption at rest with Openstack</h4>
 
     <div class="col-6">
       <h4>Tested architecture</h4>
-      <p>Our architecture around Ceph has been developed after extensive testing and production experience and utilises nearline SAS or SATA drives as bulk storage to drive lower storage costs with NVME flash as a cache device to maintain performance levels.</p>
+      <p>Our Ceph reference architectures have been developed after extensive testing and production experience. Utilising nearline SAS or SATA drives for bulk storage to drive lower storage costs with NVME flash as a cache device to maintain performance levels.</p>
```
Comment (anthonydillon) [inline templates/ceph/index.html:181]: This should be two lines it looks like

**Review by anthonydillon — DISMISSED** [review]
Few comments in the code

**Comment by minkyngkm** [comment]
Thanks @anthonydillon for reviewing! It's been all updated :)

**Review by carkod — APPROVED** [review]
LGTM

**Pull request merged**